spec/lib/integration/nation_builder_spec.rb

```diff
@@ -94,16 +94,16 @@
 
   describe "#list_counts" do
     it "formats url for target endpoint" do
-      allow(Integration::NationBuilder).to receive(:request_handler).and_call_original
+      allow(RestClient).to receive(:get).and_call_original
 
       Integration::NationBuilder.list_counts
 
-      expect(Integration::NationBuilder).to have_received(:request_handler).with(endpoint_path: Integration::NationBuilder::ENDPOINTS[:lists])
+      expect(RestClient).to have_received(:get).with(ENV['NATION_BUILDER_DOMAIN'] + Integration::NationBuilder::ENDPOINTS[:list_count_page])
```

Comment:
Line is too long. [140/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.